documentation/polymer-templates/styling-polymer-templates.asciidoc

```diff
@@ -8,11 +8,11 @@ layout: page
 
 Since client-side templates are Web Components, their content is inside shadow DOM. By design, shadow DOM defines a local style scope that is isolated from global styles. See <<../../themes/style-scopes#,Style Scopes>> for more information.
 
-== LitElement
-
 You can add component-specific scoped styles directly in the static `styles` template property.
 
-.`my-view.js`
+It is recommended to use LitElement template language for your components, but PolymerTemplate (see <<tutorial-polymer-template-basic#,PolymerTemplate, Styling Templates>>) is also supported.
```

Comment:
I would not add this as polymer template support is deprecated and to be removed